src/NServiceBus.AmazonSQS/Configure/TransportInfrastructure.cs

```diff
@@ -61,7 +61,7 @@ public override TransportReceiveInfrastructure ConfigureReceiveInfrastructure()
             return new TransportReceiveInfrastructure(
                 CreateMessagePump,
                 CreateQueueCreator,
-                () => Task.FromResult(StartupCheckResult.Success));
+                () => new RequiresWindows().Validate());
```

Comment:
Do we want to allow send-only endpoints on linux? If not, this startup check should be on `ConfigureSendInfrastructure` instead.

Reply:
As the bug is receive only, I'd say 'why not'?

Reply:
It might be an easier story to say that we just don't support it on linux at all right now vs the more complicated story of "Not supported on linux, unless it's a send-only endpoint."

Reply:
I agree but on the other hand this would at least allow for example to host ASP.NET Core with Sendonly endpoints on Linux and the backend system on NET Core with windows until resolved.

Reply:
I suggest the @Particular/aws-maintainers take a decission here on what to allow

Reply:
Assuming we're going to disable the linux tests completely wouldn't this mean that we have a "supported" feature (send only on linux) where we have zero acceptance tests covering it? 

Can we do something in the interim to cover the gap? manual smoke test with a sample etc?

Reply:
@andreasohlund yeah that is a good point but given the dispatcher logic is super simple I would not be too worried about it since we test send and receive on the full framework.